pkgs/servers/sql/mariadb/connector-c/2_3.nix

```diff
@@ -0,0 +1,6 @@
+{ callPackage, ... } @ args:
+
+callPackage ./default.nix (args // {
```

Comment:
`./.` is the standard way to call `default.nix` in current directory.

Reply:
Thanks, fixed